config/webpack.config.js

```diff
@@ -40,7 +40,10 @@ module.exports = function()
 				{
 					test: /\.ts$/,
 					exclude: /node_modules/,
-					loader: 'ts'
+					loader: 'awesome-typescript-loader',
```

Comment:
I don't mind changing this to awesome-typescript-loader, but if we use a separate webpack config for test we could theoretically still use the 'default' ts-loader here. Maybe we should test the performance of the loaders when we have a bigger test project in skeleton. For now though, let's just use the same loader for both.